css/css-properties-values-api/url-resolution.html

```diff
@@ -9,8 +9,8 @@
 <link id="main" rel="stylesheet" type="text/css" href="support/main/main.css" />
 <link id="main_utf16be" rel="stylesheet" type="text/css" href="support/main/main.utf16be.css" />
 <link id="alt" rel="stylesheet" type="text/css" href="support/alt/alt.css" />
-<div id=target>
-    <div id=inner></div>
+<div id="target">
+    <div id="inner"></div>
```

Comment:
Omitting `"` is perfectly fine, but messes with syntax highlighting for the rest of the file.